Sources/Security/Signing.swift

```diff
@@ -234,13 +237,22 @@ extension Signing {
 
 extension Signing.SignatureParameters {
 
+    @available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *)
     var asData: Data {
+        let nonce: Data = self.nonce ?? .init()
+        let path: Data = self.path.unescapedPath.asData
+        let postParameterHash: Data = self.requestBody?.postParameterHash.asData ?? .init()
+        let requestDate: Data = String(self.requestDate).asData
+        let etag: Data = (self.etag ?? "").asData
+        let message: Data = self.message ?? .init()
+
         return (
-            (self.nonce ?? .init()) +
-            self.path.unescapedPath.asData +
-            String(self.requestDate).asData +
-            (self.etag ?? "").asData +
-            (self.message ?? .init())
+            nonce +
+            path +
+            postParameterHash +
+            requestDate +
+            etag +
+            message
         )
```

Comment:
Separating this sped up compilation from 4 seconds to less than 100ms (not sure exactly how long, but that was the threshold I set for the warning).